digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1770,6 +1770,52 @@ object DataFixture {
     println("Complete at time: " + DateTime.now())
   }
 
+  def getStateRoadWithFunctionalClassOverridden(): Unit = {
+    println("\nStart process to get StateRoads With Functional Class Overridden")
+    println(DateTime.now())
+    println("")
+
+    val functionalClassValue = 5
+    val sinceDate = Some("20190101") //Format required YYYYMMDD
+
+    OracleDatabase.withDynTransaction {
+      val linkIdsOverridden = RoadLinkDAO.getLinkIdByValue("functional_class", functionalClassValue, sinceDate).toSet
```

Comment:
Since FunctionalClassDao extends from RoadLinkDAO, this should be like 
 FunctionalClassDao .getLinkIdByValue(functionalClassValue, sinceDate).toSet